instances/technique/receptiveFieldMapping.jsonld

```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/receptiveFieldMapping",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "'Receptive field mapping' is use to map out which portion of sensory space elicit neuronal responses when stimulated.",
```

Comment:
**Alternative definition:**  "'Receptive field mapping' is a stimulation technique used to map out which neuron or neuron population preferably elicit a neuronal response to a specific sensory stimulus or sensory stimulus set." 

@tgbugs I think we need your additional feedback here for a nice definition.